tests/test_rescue.py

```diff
@@ -584,85 +462,204 @@ def test_mark_for_deletion_setter_bad_data(self, reason: str or None, reporter:
-        """
-        Verifies Rescue.__eq__ raises a type error when attempting to compare something
-            other than a rescue.
-        """
-        assert not rescue_plain_fx == "Rescue object at <0xBADBEEF> "
+    await result_rescue.add_rat(name=name, guid=uuid)
 
-    @pytest.mark.parametrize("reporter, reason", [("unit_test[BOT]", "reasons! reasons i say!"),
-                                                  ("potato[pc|nd]", "uhhh..."),
-                                                  ("sayWhat99", "dawg this ain't right!")])
-    def test_mark_delete_valid(self, rescue_sop_fx: Rescue, reporter: str, reason: str):
-        """Verifies Rescue.mark functions as expected when marking a case for deletion"""
+    assert name in Rats.cache_by_name
 
-        rescue_sop_fx.mark_delete(reporter, reason)
 
-        assert rescue_sop_fx.marked_for_deletion.marked
-        assert reporter == rescue_sop_fx.marked_for_deletion.reporter
-        assert reason == rescue_sop_fx.marked_for_deletion.reason
+def test_eq_none(rescue_plain_fx: Rescue):
+    """
+    Verifies behavior of `Rescue.__eq__` when comparing against None
+    """
+    # This check only exists because this object is nullable...
+    # and no, you really shouldn't be comparing against None like this.
+    assert not None == rescue_plain_fx
 
-    def test_mark_delete_invalid(self, rescue_sop_fx: Rescue):
-        """verify what happens when garbage gets thrown at `rescue.mark`"""
-        with pytest.raises(TypeError):
-            rescue_sop_fx.mark_delete(None, "sna")
+
+def test_eq_bad_type(rescue_plain_fx: Rescue):
+    """
+    Verifies Rescue.__eq__ raises a type error when attempting to compare something
+    other than a rescue.
+    """
+    assert not rescue_plain_fx == "Rescue object at <0xBADBEEF> "
+
+
+@pytest.mark.parametrize("reporter, reason", [("unit_test[BOT]", "reasons! reasons i say!"),
+                                              ("potato[pc|nd]", "uhhh..."),
+                                              ("sayWhat99", "dawg this ain't right!")])
+def test_mark_delete_valid(rescue_sop_fx: Rescue, reporter: str, reason: str):
+    """
+    Verifies Rescue.mark functions as expected when marking a case for deletion
+    """
+
+    rescue_sop_fx.mark_delete(reporter, reason)
+
+    assert rescue_sop_fx.marked_for_deletion.marked
+    assert reporter == rescue_sop_fx.marked_for_deletion.reporter
+    assert reason == rescue_sop_fx.marked_for_deletion.reason
+
+
+def test_mark_delete_invalid(rescue_sop_fx: Rescue):
+    """
+    Verify what happens when garbage gets thrown at `rescue.mark`
+    """
+    with pytest.raises(TypeError):
+        rescue_sop_fx.mark_delete(None, "sna")
 
         with pytest.raises(TypeError):
             rescue_sop_fx.mark_delete("sna", None)
 
         with pytest.raises(ValueError):
             rescue_sop_fx.mark_delete("unit_test", "")
+            
+
+def test(rescue_sop_fx: Rescue):
```

Comment:
While this is technically a valid name and will be detected during test discovery, can we please rename this to describe what the test actually is? ((based on the git diff methinks i might be to blame for this test's name... 😭 ))

Reply:
renamed :)